plug.vim

```diff
@@ -600,13 +600,24 @@ function! s:update(force, names)
   call s:update_impl(1, a:force, a:names)
 endfunction
 
+function! s:is_exec_helptags(docdir)
```

Comment:
This function name is un-clear to be what should do.

Reply:
How about `s:is_need_exec_helptags` ?

Reply:
Since I'm not native speaker, I cannot answer best. But has_helptags(x) or helptags_exists(x) ?
cc @junegunn

Reply:
I think `do_need_exec_helptags()` is best.
I'll change it after I came back today or tomorrow. :kissing_smiling_eyes:

Thank you mattn :+1: